internal/http/services/owncloud/ocs/handlers/apps/sharing/sharees/sharees.go

```diff
@@ -100,10 +100,15 @@ func (h *Handler) FindSharees(w http.ResponseWriter, r *http.Request) {
 }
 
 func (h *Handler) userAsMatch(u *userpb.User) *conversions.MatchData {
+	var userType int
+	if u.Id.Type == userpb.UserType_USER_TYPE_GUEST || u.Id.Type == userpb.UserType_USER_TYPE_LIGHTWEIGHT {
+		userType = 1
+	}
+
 	return &conversions.MatchData{
 		Label: u.DisplayName,
 		Value: &conversions.MatchValueData{
-			ShareType: int(conversions.ShareTypeUser),
+			ShareType: userType,
```

Comment:
I find this a bit confusing here, there is a fork from @c0rby here that defines a struct for usertypes https://github.com/C0rby/go-cs3apis/blob/main/cs3/identity/user/v1beta1/resources.pb.go#L27-L45 that seems to be still consumend (though not consistent with https://github.com/cs3org/go-cs3apis)... can we resolve that or use a different variable name here?

Reply:
different API, this is a custom ocs property.

Changed the naming and added comment.